src/plane.ts

```diff
@@ -10,29 +16,25 @@ export class Plane {
   chord: number;
   model: THREE.Group;
 
-  //flightTensor: [[number]];
-  constructor() {
-    //this.flightTensor = [
-    //]
-    this.wingspan = 0.9;
-    this.length = 0.6;
-    this.chord = 0.08;
-    const boxGeo = new THREE.BoxGeometry(
-      this.wingspan,
-      this.length,
-      this.length * this.chord
-    );
-    const coneGeo = new THREE.ConeGeometry(
-      this.wingspan * 0.1,
-      this.chord * 1.1,
-      50,
-      16
-    );
+  constructor(wingspan: number, length: number, chord: number) {
+    this.wingspan = wingspan;
+    this.length = length;
+    this.chord = chord;
+    this.model = this.genStandinGeometry(wingspan, length, chord);
+  }
+  genStandinGeometry(
+    wingspan: number,
+    length: number,
+    chord: number
+  ): THREE.Group {
+    const boxGeo = new THREE.BoxGeometry(wingspan, length, length * chord);
+    const coneGeo = new THREE.ConeGeometry(wingspan * 0.1, chord * 1.1, 50, 1);
     const mat = new THREE.MeshNormalMaterial();
```

Comment:
If this is the long-term material setup, it'd be more readable to define this as a constant property, typically using `private static readonly`:

`private static readonly MATERIAL = new THREE.MeshNormalMaterial();`

Then you could refer to it as `Plane.MATERIAL` when using it.

I say this is more readable because [the all-caps naming is reserved for constants in the Google style guide](https://google.github.io/styleguide/tsguide.html#identifiers-constants), so a reader who sees `Plane.MATERIAL` will be able to tell it's a `static readonly` property of `Plane` without needing to look for where it's defined. And because they know it's `readonly`, they don't have to worry about needing to track how `mat` might change over the course of the function. In long functions*, a lot of readability comes from making it clear to the reader which variables they need to keep track of.

*not that this is a long function, I'm just being extra-explainy

Reply:
Seems reasonable. I'll eventually be changing from MeshNormalMaterial, so this'll make it easier to find as well.